docs/installation/prepare/jetson-nano-setup.md

```diff
@@ -66,6 +66,29 @@ If this command fails, try using `wget https://storage.googleapis.com/packages.v
 
 {{< readfile "/static/include/install/install-linux-aarch.md" >}}
 
+## Serial Communication Protocol Tips
+
+To change the pins that are in use for modes of serial communication, launch <file>jetson-io.py</file> with the following command:
+
+```sh { class="command-line" data-prompt="$"}
+sudo /opt/nvidia/jetson-io/jetson-io.py
+```
+
+In the interactive menu that opens, select **Configure Jetson 40 Pin Header** and **Configure header pins manually** to select and deselect pins to enable use.
+For a Jetson Orin Nano, reference the following:
+
+<!-- prettier-ignore -->
+| GPIO Header Pin | Viam Bus ID | `jetson-io.py` ID |
+| ---------------| ----------- | ----------------- |
+| 3, 5 | `7` | `i2c2` |
+| 27, 28 | `1` | `i2c8` |
+| 19, 21, 23, 24, 26 | `0` | `spi1` |
+| 15 | | `pwm1` |
+| 33 | | `pwm5` |
+| 32 | | `pwm7` |
```

Comment:
I've never gotten hardware PWM to work on pin 32. So, the RDK uses hardware PWM only on pins 15 and 33. I agree that pin 32 claims to have it: I don't know what I've been doing wrong on that one. but enabling it won't (currently) make the RDK use it.

Reply:
Again, for your reference, this is copied straight from `jetson-io.py` on an Orin Nano:
```
 |                Select desired functions (for pins):                |
 |                                                                    |
 |                [ ] aud            (7)                              |
 |                [ ] extperiph3_clk (29)                             |
 |                [ ] extperiph4_clk (31)                             |
 |                [ ] i2s2           (12,35,38,40)                    |
 |                [*] pwm1           (15)                             |
 |                [*] pwm5           (33)                             |
 |                [*] pwm7           (32)                             |
 |                [ ] spi1           (19,21,23,24,26)                 |
 |                [ ] spi3           (13,16,18,22,37)                 |
 |                [ ] uarta-cts/rts  (11,36)                          |
```